ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__arm64-stable-4.14-upgrade-from-stable-4.13.yaml

```diff
@@ -86,6 +86,20 @@ tests:
     test:
     - chain: openshift-upgrade-qe-sanity-disconnected
     workflow: cucushift-installer-rehearse-aws-ipi-disconnected-private-cco-manual-security-token-service-private-s3-with-ep-sts-ec2-elb
+- as: aws-ipi-usertags-custom-sg-f28
```

Comment:
For stable-stable upgrade, we are aim to make sure payloads with signature can be upgraded, so don't need to cover all profiles. so following this rule, you just need to check here we have covered all platforms, you don't need to add extra jobs.